.travis.yml

```diff
@@ -5,13 +5,20 @@ language: scala
 
 before_install:
   # using jabba for custom jdk management
-  - curl -sL https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh | bash && . ~/.jabba/jabba.sh
-  - jabba install adopt@1.8.202-08
-  - java -version
+  - curl -sL https://git.io/jabba | bash && . ~/.jabba/jabba.sh
+  - jabba install adopt@1.8.212-03
+  - jabba install adopt@1.11.0-3
```

Comment:
You could possibly use the `~` notation to use the latest build of the JDK versions.

Reply:
Not working right now for `adopt@~1.11`. I assume jabba is failing to correct check versions such as `adopt@1.11.0-2` because of the `-2` section. Anyway, should not block this PR.